examples/Forms/MultiLineTextInput.md

```diff
@@ -0,0 +1,159 @@
+Basic multi-line text box
+
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('CS50 is an on-campus and online introductory course on computer science taught at Harvard University and Yale University.');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with an error
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  errorMessage="Please provide a value for this field"
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with a hidden error
```

Comment:
What would be the use case when we want to hide an error?

Reply:
Actually not sure. I added it for consistency with `TextInput`. It might make sense for it to be removed, though. Done in 9bcb05883e5eaf6d6aecb5b8a852a90c0523c85f